sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -543,6 +544,15 @@ def test_Beam3D():
     b3.solve_for_reaction_loads(R1, R2, R3, R4)
     assert b3.reaction_loads == {R1: -120, R2: -120, R3: -1350, R4: -2700}
 
+def test_polar_moment_Beam3D():
+    l, E, G, A, I1, I2 = symbols('l, E, G, A, I1, I2')
+    R1, R2, R3, R4 = symbols('R1, R2, R3, R4')
+    I=[I1, I2]
+
+    b = Beam3D(l, E, G, I, A)
+    assert b.polar_moment() == I1 + I2
+
+    assert b.polar_moment('x') == I1 + I2
```

Comment:
Still need to test for `p.polar_moment('y')` (or some other case with non-valid inputs). It may make sense to raise an error rather than just not returning anything.

Reply:
So should I remove the axis part or raise an error?

Reply:
Since the beam can have any arbitrary cross section and we don't really know its dimensions, it'll be better if I remove the axis part. Moreover, for calculations we usually use Polar moment about axis along the beam.

Reply:
There is no need to calculate polar moments about any other axis than X in the class.